extensions/cl_intel_unified_shared_memory.asciidoc

```diff
@@ -231,12 +231,13 @@ Optional allocation properties may also be queried using *clGetMemAllocInfoINTEL
 ----
 typedef cl_uint cl_mem_info_intel;
 
-#define CL_MEM_ALLOC_TYPE_INTEL         0x419A
-#define CL_MEM_ALLOC_BASE_PTR_INTEL     0x419B
-#define CL_MEM_ALLOC_SIZE_INTEL         0x419C
-#define CL_MEM_ALLOC_DEVICE_INTEL       0x419D
+#define CL_MEM_ALLOC_TYPE_INTEL               0x419A
+#define CL_MEM_ALLOC_BASE_PTR_INTEL           0x419B
+#define CL_MEM_ALLOC_SIZE_INTEL               0x419C
+#define CL_MEM_ALLOC_DEVICE_INTEL             0x419D
+#define CL_MEM_ALLOC_BUFFER_LOCATION_INTEL    0x419E
```

Comment:
I'm fine saying that the USM extension interacts with the new `cl_intel_mem_alloc_buffer_location` extension (see the [section](https://github.com/KhronosGroup/OpenCL-Docs/blob/master/extensions/cl_extension_template.asciidoc#interactions-with-other-extensions) of the extension template that describes how to do this), but I don't think that this enum should be listed directly in the USM spec.  Many USM implementations will not support the buffer location feature.

Reply:
This change still needs to be reverted, and an "Interactions with Other Extensions" could be added in its place.

Reply:
I have removed all reference of buffer_location, and moved some info under interaction section. Sorry I originally misunderstood it to be adding "interaction" to buffer_location extension doc (the interaction section is still present in buffer_location doc).